core/.vscode/settings.json

```diff
@@ -36,8 +36,7 @@
     "vitejs"
   ],
   "files.exclude": {
-    "node_modules": true,
-    "dist": true
+    "node_modules": true
```

Comment:
Is this because the core build ends up in dist in dev mode?

Reply:
This is just so I can actually see when its built because I kept wondering why it wasn't working and then I'd realize I didn't have a dist. This can probably be reverted later.

Reply:
oohh ok, got it